packages/form/__tests__/form.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { mount } from '@vue/test-utils'
+import Form from '../src/form.vue'
+
+const AXIOM = 'Rem is the best girl'
+
+describe('Form.vue', () => {
+  test('render test', () => {
+    const wrapper = mount(Form, {
+      slots: {
+        default: AXIOM,
+      },
+    })
+    expect(wrapper.text()).toEqual(AXIOM)
+  })
+})
```

Comment:
You need to add test files before asking for review

Reply:
test in Element project `test/unit/specs/form.spec.js`, might be some compoments missing at this time

Reply:
OK i will finish it. 😊